recipes/libalsa/all/conanfile.py

```diff
@@ -72,3 +75,5 @@ def package_info(self):
         self.cpp_info.names['pkg_config'] = 'alsa'
         self.cpp_info.names["cmake_find_package"] = "ALSA"
         self.cpp_info.names["cmake_find_package_multi"] = "ALSA"
+        self.env_info.ALSA_CONFIG_DIR = os.path.join(self.package_folder, "res", "alsa")
+        self.env_info.ALSA_CONFIG_PATH = os.path.join(self.package_folder, "res", "alsa")
```

Comment:
I took another look at the sources, and `ALSA_CONFIG_PATH` needs to be a path to the `alsa.conf` file.
But when not set, it is calculated relatively to `ALSA_CONFIG_DIR`.
So it does not have to best explicitly.